app/Services/PostService.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace App\Services;
+
+use Illuminate\Support\Facades\Storage;
+use Illuminate\Support\Facades\Hash;
+use App\Models\Post;
+
+class PostService
+{
+    public function create($request)
+    {
+        return Post::create([
+            'link' => $request->input('link') ? $request->input('link') : '',
```

Comment:
Default value
```suggestion
            'link' => $request->input('link', ''),
```

Hoặc short ternary
```suggestion
            'link' => $request->input('link') ?: '',
```

Reply:
(votay)